framework/web/AssetManager.php

```diff
@@ -532,12 +553,8 @@ public function getPublishedPath($path)
             return $this->_published[$path][0];
         }
         if (is_string($path) && ($path = realpath($path)) !== false) {
-            $base = $this->basePath . DIRECTORY_SEPARATOR;
-            if (is_file($path)) {
-                return $base . $this->hash(dirname($path) . filemtime($path)) . DIRECTORY_SEPARATOR . basename($path);
-            } else {
-                return $base . $this->hash($path . filemtime($path));
-            }
+            return $this->basePath . DIRECTORY_SEPARATOR . $this->hash($path) . 
```

Comment:
I am usure splitting in 2 lines makes sense here: it would be better to write in single one - it whould be long then former line number 537.